### DISCUSSION TITLE
Can we we have an api that can be used on chatbot UI

### DISCUSSION THREAD

**Discussion by nrajukrishna** [RamiAwar/dataline#257] [Q&A] [answered]
Can we we have an api that can be used on chatbot UI , please advise how to do this to make this as api service

**Comment by RamiAwar** [comment]
DataLine is already a backend + frontend so you could technically just use the backend. But we haven't built it with this in mind so far, as in we're not offering first-class headless support. We're thinking of making that pivot tho.

But you could already figure out how to use it, the app is fully API driven!

**Comment by anthony2261** [comment, marked as answer]
If you manage to run the app, you could check out the API specs via http://localhost:7377/docs
